pages/home.py

```diff
@@ -16,6 +16,8 @@ class HomePage(Base):
     _displayed_profile_name_locator = (By.CSS_SELECTOR, '.billboard h3')
     _task_in_progress_locator = (By.ID, 'task-in-progress')
     _pick_a_task_locator = (By.ID, 'pick-a-task')
+    _displayed_suggested_first_tasks_list_locator = (By.CSS_SELECTOR, '.task-list-container h3')
```

Comment:
This is the heading for the task list, so perhaps a better name would be something like `suggested_first_tasks_heading_locator`. We shouldn't include the word `displayed` in the name of the locator as we may use this locator to find an element that is not currently displayed.